mlflow/store/artifact/azure_blob_artifact_repo.py

```diff
@@ -178,3 +197,62 @@ def _download_file(self, remote_file_path, local_path):
 
     def delete_artifacts(self, artifact_path=None):
         raise MlflowException("Not implemented yet")
+
+    def create_multipart_upload(self, local_file, num_parts=1, artifact_path=None):
+        from azure.storage.blob import BlobSasPermissions, generate_blob_sas
+
+        (container, _, dest_path, _) = self.parse_wasbs_uri(self.artifact_uri)
+        if artifact_path:
+            dest_path = posixpath.join(dest_path, artifact_path)
+        dest_path = posixpath.join(dest_path, os.path.basename(local_file))
+
+        # Put Block: https://learn.microsoft.com/en-us/rest/api/storageservices/put-block?tabs=microsoft-entra-id
+        # SDK: https://learn.microsoft.com/en-us/python/api/azure-storage-blob/azure.storage.blob.blobclient?view=azure-python#azure-storage-blob-blobclient-stage-block
+        blob_url = posixpath.join(self.client.url, container, dest_path)
+        sas_token = generate_blob_sas(
+            account_name=self.client.account_name,
+            container_name=container,
+            blob_name=dest_path,
+            account_key=self.client.credential.account_key,
+            permission=BlobSasPermissions(read=True, write=True),
+            expiry=datetime.datetime.utcnow() + datetime.timedelta(hours=6),
```

Comment:
Curious about the expiration time - is this something that Azure recommends for duration?

Reply:
I think I misread the documentation somewhere. Let me change to 1 hour to align with our s3 & gcs impl